src/services/db/RepoService.ts

```diff
@@ -38,27 +38,19 @@ export default class RepoService extends GitHubService {
     this.gitFileSystemService = gitFileSystemService
   }
 
-  getGgsWhitelistedRepos(
-    growthbook: GrowthBook<FeatureFlags> | undefined
-  ): string[] {
-    if (!growthbook) return []
-
-    const whitelistedRepos = growthbook.getFeatureValue(
-      FEATURE_FLAGS.GGS_WHITELISTED_REPOS,
-      { repos: [] }
-    )
-    return whitelistedRepos.repos
-  }
-
-  isRepoWhitelisted(repoName: string, ggsWhitelistedRepos: string[]): boolean {
+  isRepoWhitelisted(
+    repoName: string,
+    growthbook?: GrowthBook<FeatureFlags>
+  ): boolean {
     // TODO: Adding for initial debugging if required. Remove once stabilised
-    logger.info(
-      `Evaluating if ${repoName} is GGS whitelisted: ${ggsWhitelistedRepos.includes(
-        repoName
-      )}`
+    logger.info(`Evaluating if ${repoName} is GGS whitelisted.`)
+    // NOTE: Growthbook has to be optional
+    // as we cannot guarantee its presence on `sessionData`.
+    // It is present iff there is a `siteHandler` attached.
+    return !!growthbook?.getFeatureValue(
+      FEATURE_FLAGS.IS_GGS_WHITELISTED,
```

Comment:
are we using the wrong feature fflag here?

Reply:
actually no, i just named it wrongly cos i think i got confused between white/blacklist